FEATURES.md

```diff
@@ -134,8 +166,10 @@
     - *stage*
     - *Location*
 - **Ender Nodes Helper**
+- **Special Effects for rare dye drops**
```

Comment:
should go with the dungeon drops one

Reply:
so remove the **Rare Drop Special Effects** from Dungeon features and adding this to Visual Enhancements

 - **Rare Drop Special Effects**
   - Special dye
   - Special dungeon items
   
 ?

Reply:
probably fine then, I thought it was organized like how it is in the config (in general)